index.html

```diff
@@ -50,7 +53,179 @@ <h2>Software Developer</h2>
         need coded, don’t hestiate to contact me.
       </p>
     </section>
+    <!--headline sections ends here-->
 
-    <hr id="horizontal-line" />
+    <hr class="horizontal-line" />
+
+    <!--about sections-->
+    <section class="about-section">
+      <div>
+        <h3 class="title">Languages</h3>
+        <ul class="skills">
+          <li>Javascript</li>
+          <li>Python</li>
+          <li>Ruby</li>
+          <li>Html</li>
+          <li>Css</li>
+        </ul>
+      </div>
+
+      <div>
+        <h3 class="title">Frameworks</h3>
+        <ul class="skills">
+          <li>Bootstrap</li>
+          <li>Ruby on rails</li>
+          <li>RSpec</li>
+          <li>CapyBara</li>
+          <li>Selenium</li>
+        </ul>
+      </div>
+
+      <div>
+        <h3 class="title">Skills</h3>
+        <ul class="skills">
+          <li>Codekit</li>
+          <li>Github</li>
+          <li>Codepen</li>
+          <li>Gitlab</li>
+          <li>Terminal</li>
+        </ul>
+      </div>
+    </section>
+    <!--about sections ends here-->
+
+    <hr class="horizontal-line" />
+
+    <!--section for social links and icons-->
+    <!--webhint flags section tags without heading as error.
+    Hence div tag is used to wrap the icons to pass the webhint check.
+    This approach is used since there is no heading given in the figma design on figma-->
+    <div class="social-icons">
+      <a href="#"><i class="fab fa-github"></i></a>
+      <a href="#"><i class="fab fa-linkedin-in"></i></a>
+      <a href="#"><i class="fab fa-instagram"></i></a>
+      <a href="#"><i class="fab fa-twitter"></i></a>
+      <a href="#"><i class="fab fa-facebook"></i></a>
+    </div>
```

Comment:
- [ ] The list of icons has a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Please change it.

Reply:
done